src/plugins/kibana_overview/public/components/_overview.scss

```diff
@@ -1,14 +1,3 @@
-.kbnOverviewApps__item,
-.kbnOverviewMore__item {
-  // Ensure card heights are stretched equally when wrapped with this element
-  .kbnRedirectCrossAppLinks {
-    align-items: flex-start;
-    display: flex;
-    flex: 1;
-    flex-direction: column;
-  }
-}
-
```

Comment:
Deleted since this declaration will not target any element in any possible combination, given this class is removed here https://github.com/elastic/kibana/pull/170304/files#diff-869c406edea4416caab5e20945eaa147bf14e5757206b11c0e8269a59054bf79L65